Topics/System_Design/DesignPatterns/Creational_Design_Patterns.md

```diff
@@ -0,0 +1,126 @@
+# Creational Design Patterns
+
```

Comment:
Adding a table of contents for easy navigation and overview of content on this page could be a good addition.